lib/eventually_helper.rb

```diff
@@ -15,4 +15,9 @@ def eventually(options = {})
       sleep interval
     end
   end
+
+  def eventually_expect &assertion
+    eventually{raise unless assertion.call}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.